src/python/com/illinoistriangle/bot/outline_bot.py

```diff
@@ -72,39 +70,81 @@ def lambda_handler(data, context):
   handle_event(slack_event)
 
 
-def handle_event(e):
-  if 'subtype' in e and e['subtype'] == VALUE_TYPE_BOT_MESSAGE:
-    return
+def search_thread_parent_for_urls(channel, thread_ts):
+  """return list of urls from thread's parent post"""
+  #TODO enable scope channels:history
+  response = sc.api_call(
+              'channels.replies',
+              channel=channel,
+              thread_ts=thread_ts,
+              )
+
+  urls = []
+  if response['ok'] == 'true' and 'messages' in response:
+    # from all thread messages, find earliest ts and return the text of that post
+    parent_thread_text = sorted([x['text'] for x in response['messages']], key=lambda x: x['ts'])[0]
+
+    urls = find_urls(parent_thread_text)
```

Comment:
probably
```
urls.extend(find_urls(parent_thread_text))
```
unless you want to overwrite `urls` every iteration

Reply:
ok done